VSDiagnostics/VSDiagnostics/VSDiagnostics/Utilities/Extensions.cs

```diff
@@ -10,9 +10,10 @@
 
 namespace VSDiagnostics.Utilities
 {
+    // TODO: after null checks, throw argumentnullexceptions instead of returning false
     public static class Extensions
     {
-        private static Dictionary<string, string> _aliasMapping = new Dictionary<string, string>
+        private static readonly Dictionary<string, string> AliasMapping = new Dictionary<string, string>
             {
                 {nameof(Int16), "short"},
```

Comment:
Hah. I was going to create a new issue and it looks like I've already came across this once: https://github.com/Vannevelj/VSDiagnostics/issues/269